pandas/tests/test_testing.py

```diff
@@ -215,6 +215,13 @@ def test_multiindex_dtype(self):
                 {'a':[1.0,2.0],'b':[2.1,1.5],'c':['l1','l2']}, index=['a','b'])
         self._assert_not_equal(df1, df2, check_index_type=True)
 
+    def test_empty_dtypes(self):
```

Comment:
I think this is the right place (TestAssertFrameEqual) for a regression test of assert_frame_equals, which is what this is... ?